projects/Amlogic/linux/linux.aarch64.conf

```diff
@@ -404,7 +404,7 @@ CONFIG_PM_SLEEP=y
 CONFIG_PM_SLEEP_SMP=y
 # CONFIG_PM_AUTOSLEEP is not set
 # CONFIG_PM_WAKELOCKS is not set
-# CONFIG_PM_RUNTIME is not set
+CONFIG_PM_RUNTIME=y
```

Comment:
Why is this change needed? IIRC this was turned off because of some issues with USB devices.

Reply:
This is preparing for crazycat